base/src/androidLibMain/kotlin/text/NumberFormatter.kt

```diff
@@ -0,0 +1,144 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+ 
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+ 
+      http://www.apache.org/licenses/LICENSE-2.0
+ 
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  
+ */
+
+package com.splendo.kaluga.base.text
+
+import com.splendo.kaluga.base.utils.Locale
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import java.util.Currency
+
+@ExperimentalUnsignedTypes
+actual class NumberFormatter actual constructor(actual val locale: Locale, style: NumberFormatStyle) {
+
+    private val format: DecimalFormat = when (style) {
+        is NumberFormatStyle.Integer -> DecimalFormat.getInstance(locale.locale).apply {
+            minimumIntegerDigits = style.minInteger.toInt()
```

Comment:
This looks strange, minimum integer digits as amount of digits to show. But `minInteger` sounds like `Integer.MIN_VALUE`, which is negative. I think we need a better name of number style properties

Reply:
renamed al min/max variables to min/maxXDigits